bigtable/google/cloud/bigtable/row_data.py

```diff
@@ -376,6 +376,9 @@ def __init__(self, read_method, request, retry=DEFAULT_RETRY_READ_ROWS):
         self.rows = {}
         self._state = self.STATE_NEW_ROW
 
+        # Flag to stop iteration, for any reason not related to self.retry()
+        self._stop = False
```

Comment:
Looking over the code, it seems this is marking if the iterator has been cancelled. The comment and name of this sort of directed me to think maybe this did something different. `self._cancelled` be accurate?

Reply:
+1 for `_cancelled`

Reply:
Makes sense. Done.